README.md

```diff
@@ -36,8 +36,24 @@ The easiest way to build daal4py is using the conda-build with the provided reci
 * Python version >= 3.6
 * conda-build version >= 3
 * C++ compiler with C++11 support
+For oneAPI support:
+* A DPC++ compiler 
+* Intel(R) oneAPI Data Analytics Library version 2021.1 or later (https://github.com/01org/daal)
+  * You can use the pre-built conda package from Intel channel on anaconda.org (see below)
 
 ## Building daal4py
+For build with oneAPI support additional step requered:
+1. DPC++ compiler - DPCPPROOT variable should be defiend and pointed to DPCPP compier
+2. Intel(R) oneDAL with oneAPI support
+    a. Can be installed from Conda channel
+    b. Can be installed fron oneAPI packages repository and passed via DALROOT variable
```

Comment:
also maybe add a link to conda?

not clear: if installed with conda, DALROOT is not needed?

```suggestion
2. Install Intel(R) oneAPI Data Analytics Library with oneAPI support:

    a. From Conda channel.
    b. From oneAPI packages repository (pass the path to oneDAL via DALROOT variable)
```